packages/shared/src/common/date.ts

```diff
@@ -11,8 +12,30 @@ export function isValidISODate(date: string): boolean {
   return buildDayjs(date, ISO_DATE, true).isValid();
 }
 
-const isValidISODateOptional = (date: string | undefined | null): boolean =>
-  date ? isValidISODate(date) : true;
+function isValidISODateOptional(date: string | undefined | null): boolean {
+  return date ? isValidISODate(date) : true;
+}
+
+export function validateIsPast(date: string): boolean {
+  if (dayjs(date).isAfter(dayjs())) {
+    throw new BadRequestError(`Date must be in the past`, undefined, { date });
```

Comment:
@leite08 if it can be the current date then this should probably be `Date can't be in the future`

Reply:
You could argue the name of the method should be `validateIsPastOrCurrent` as well if this is the case.